Appium/generate_config.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+if [ -z "$PLATFORM"]; then
+  PLATFORM="Android"
```

Comment:
BTW, if we implement getting some device info via adb command, do we even need option for platformName or maybe it can be set by default to `Android` since we kind of assume it will be Android?
I don't have much experience, almost none actually :D, with apple ecosystem, does this grid functionality even have potential application and support for IOS devices?